web/templates/pages/member_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}TrackCat Member{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="member_profile">
+			<div class="col-md-12">
+			<h2>{% if user.fullname %}
+			{{ user.fullname }} 
+			{% else %}
+			{{ user.username }}
+			{% endif %}'s profile</h2>
+			{% for field in form %}
```

Comment:
Yes and how exactly should errors display? 
I don't think this portion is understood so much, so it is better to ask someone before sending PR.